src/components/common/DetailComment.vue

```diff
@@ -1,17 +1,19 @@
 <template>
   <v-card flat id="detail-comment-container">
-    <v-textarea
-      ref="textarea"
-      auto-grow
-      rows="5"
-      id="detail-comment-textarea"
-      :counter="maxLength"
-      :rules="rules"
-      :value="value"
-      :placeholder="placeholder"
-      :autofocus="autofocus"
-      @input="emitInput($event)"
-    />
+    <v-row no-gutters class="pl-4 pr-4 pt-2 pb-2">
+      <v-textarea
+        ref="textarea"
+        auto-grow
+        rows="5"
+        id="detail-comment-textarea"
+        :counter="maxLength"
+        :rules="rules"
+        :value="value"
+        :placeholder="placeholder"
+        :autofocus="autofocus"
+        @input="emitInput($event)"
+      />
+    </v-row>
```

Comment:
Thanks for the great description of changes in the PR 👍 

Vuetify renders a v-row as a div with some classes, so you're right -- underneath, they're equivalent.

In this case, I recommend a div instead, since you're not actually using any row functionality (such as columns within it).

With a div, you may have to explicitly add some Vuetify classes that "row" automatically adds, like flexbox. Ref: https://vuetifyjs.com/en/styles/flex/

Reply:
That makes sense, I'll switch it to a `div`. I did a quick test comparing `v-row` and `div` and didn't notice any immediate differences but I'll double check to make sure that there are no Vuetify classes for `v-row` that need to be added.